include/deal.II/base/qprojector.h

```diff
@@ -136,7 +136,7 @@ class QProjector
   project_to_face(const ReferenceCell               &reference_cell,
                   const SubQuadrature               &quadrature,
                   const unsigned int                 face_no,
-                  const types::geometric_orientation orientation);
+                  const types::geometric_orientation combined_orientation);
```

Comment:
I wonder whether it would be worth to make "Combined orientation" an entry in the glossary because it now appears so often.

Reply:
I started rewriting the entry on the face orientation but its really the wrong approach - AFAICT it dates to the time when the library only supported `face_orientation` but not `face_rotation` or `face_flip`. Do you mind if I tackle that in a follow-up? There are a lot of places which will need to be updated.

Reply:
Of course. BTW, I don't particularly mind if we have glossary entries for technical terms that use different words but have overlapping meaning. Let's make it easy for people to find what a specific term means. Individual entries in the glossary can then always reference each other, or a common page such as a module.